dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileSetReader.cpp

```diff
@@ -207,14 +194,6 @@ size_t ColumnFileSetReader::readRows(
         }
     }
 
-    if (auto delta_bytes = columnsSize(output_columns) - bytes_before_read; delta_bytes > 0)
-    {
-        if (row_ids == nullptr)
-            lac_bytes_collector.collect(delta_bytes);
-        if (likely(context.scan_context))
-            context.scan_context->user_read_bytes += delta_bytes;
-    }
```

Comment:
Where do we add the bytes read of delta layer to the `user_read_bytes` after these code changes?

Reply:
ConcatSkippableBlockInputStream